### PR TITLE
Features

### PR DESCRIPTION
**Pull request by ricardo123321**
In this project, we emulated a simple to-do list page making use of DOM manipulation elements allowing the transitions in the page to be dynamic. Some important features are: 

- Boostrap Styling for Menu Items.

- Separate Module for Display.

- Github Pages Live Link

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @ricardo123321 and @JohannRodriguez 
### You have done well so far :clap: However, you have a few issues to fix:-
- [x] Install all the required dependencies for linting and styling, hence run these commands `npx eslint .` and `npx stylelint "**/*.{css,scss}"` If there errors, kindly solve them
- [x] In the scripts in package.json file add more scripts. See below
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```
- [x] Improve the design, see more from the templates given in the instructions
- [x] We recommend you use es6 function syntax
### Make all the changes and submit for another code review. Good luck.
- Happy coding!

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+function Project(name) {
+  this.name = name;
+  this.todoItems = [];
+}
+
+function TodoIs(name, projectN, description, dueDate, priority) {
+  this.name = name;
+  this.projectN = projectN;
+}
+
```
Comment (rOluochKe) [inline src/index.js]: ### Use es6 syntax

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+submitBtnErz.addEventListener('click', () => {
+  const projectT = document.getElementById('projectT').value;
+  deleteP(projectT);
+});
+
+createP('Default');
+
+doma();
+selectobj();
```
Comment (rOluochKe) [inline src/index.js]: ### Separate logic from dom manipulation

**Review by christianotieno — CHANGES_REQUESTED** [review]
# Changes Required ♻️ 
Hello @ricardo123321  & @JohannRodriguez 👋 
Good job so far. Please, we still have some issues to resolve before moving forward. 🚧 

### Expected changes 👇 
- [ ] There should be a 'default' project awaiting to-dos creations when the page loads. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/projects_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list)).

### Optional 
- [ ] Can our forms used to create a project or a todo have validations, where you should not be able to create an empty project/todo?

### Happy coding 💻 

#### Christian Otieno (TSE)

**Review by rOluochKe — APPROVED** [review]
Hello Team!
### You got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**